docs/tutorials/google/compare_to_calibration.ipynb

```diff
@@ -0,0 +1,760 @@
+{
```

Comment:
<p>Can we link to the specific section of the theory tutorial instead of linking to the whole of it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>Done! Note the explanation appears in a comment under the first cell of the [fidelities section](https://quantumai.google/cirq/qcvv/xeb_theory#fidelities).</p>